frontend-react/src/hooks/UseFileHandler.ts

```diff
@@ -11,6 +11,7 @@ export enum ErrorType {
     FILE = "file",
 }
 
+// TODO: consolidate with Format in TemporarySettingsAPITypes.ts
```

Comment:
Will add a ticket for this -- should be pretty simple, but I'd rather break it out into a separate change so as to not adversely affect the validator logic.

Reply:
https://app.zenhub.com/workspaces/web-experience-607d9d5e68b95200150fec37/issues/gh/cdcgov/prime-reportstream/8606